### ISSUE TITLE
Configure awesome lint, and require passing for PR to merge

### ISSUE THREAD

**Issue by nuke-web3** [Polkadot-Blockchain-Academy/pba-awesome#1]
https://github.com/substrate-developer-hub/awesome-substrate/tree/master/.github/workflows perhaps would be fine? 

Plus settings to enforce a pass for all PRs to merge

**Comment by niklasp** [comment]
Thanks for the setup, sounds good to start with. never heard of awesome list before but if you have good experience let's go for it. No requirements to install/test except being able to run `npx` locally i guess?

**Comment by nuke-web3** [comment]
Yeah it's a great way to get a view on various (mostly OSS) projects! https://github.com/sindresorhus/awesome#readme

https://awesomeopensource.com/ and others scrape this data if it's linted I think... so it get picked up and discoverable around more widely!
https://awesomeopensource.com/search?q=substrate for example shows a TON of great stuff :wink: 

`npx` to run the _exact_ same thing as the CI would be needed to check locally - highly suggested for sure and defined in a CONTRIBUTING.md doc probably best?

**Comment by TomaszWaszczyk** [comment]
@NukeManDan do not know what is the culture here for selection tasks, but if there is no one who wants to implement the task/workflow, I can do it. I suggest also to select someone who will be responsible for the page (up-to-date, quality etc. etc.).

Polkadot ecosystem is hard to follow because there is many places with not up to date information - creation a place with valid links is a good idea.

**Comment by nuke-web3** [comment]
If you want to help own this, then by all means go for it! I agree - there _must_ be a CODE OWNERS doc that clearly defines and automatically assigns to review PRs to the people committed to ensure this resource is maintained.
